apps/indexer/ponder.schema.ts

```diff
@@ -94,7 +96,7 @@ export const votingPowerHistory = onchainTable(
```

Comment:
The primary key definition is problematic. If `transactionHash` can be null (changed from `notNull()` to nullable), but it's still used as part of the primary key, this will cause database constraint violations when null values are inserted.